### PR TITLE
feat(blocknative): added blockprices endpoint for legacy & EIP1559

### PR DESCRIPTION
**Pull request by vnavascues**
**Current implementation**
1 Endpoint:
- `blockprices` (optional param): requires `type` (`legacy` or `eip1559`) and `confidenceLevel` params (integer from 1 to 99, both included). Type `legacy` returns an integer (wei) to be consumed as `uint`. Type `eip1559` returns a string to be consumer as `bytes32`.

**Previous implementation**
2 Endpoints:
- `blockprices-legacy`: requires `confidenceLevel` param (integer from 1 to 99, both included). Returns an integer (wei) to be consumed as `uint`.
- `blockprices-eip1559`: requires `confidenceLevel` param (integer from 1 to 99, both included). Returns a string to be consumer as `bytes32`.

### CODE REVIEW

File: packages/sources/blocknative/test/unit/blockprices-eip1559.test.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  
+    describe('validation error', () => {
+      const requests = [
+        {
+          name: 'confidenceLevel is not a number',
+          testData: {
+            id: jobID,
+            data: {
+              endpoint: 'blockprices-eip1559',
+              season: 'LinkPool'
```
Comment (1marcghannam) [inline packages/sources/blocknative/test/unit/blockprices-eip1559.test.ts:19]: Season needs to be confidenceLevel instead.

Comment (1marcghannam) [inline packages/sources/blocknative/test/unit/blockprices-eip1559.test.ts:19]: Season needs to be confidenceLevel.

File: packages/sources/blocknative/test/unit/blockprices-legacy.test.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  
+    describe('validation error', () => {
+      const requests = [
+        {
+          name: 'confidenceLevel is not a number',
+          testData: {
+            id: jobID,
+            data: {
+              endpoint: 'blockprices-legacy',
+              season: 'LinkPool'
```
Comment (1marcghannam) [inline packages/sources/blocknative/test/unit/blockprices-legacy.test.ts:19]: Season needs to be confidenceLevel.

**Review by 1marcghannam — CHANGES_REQUESTED** [review]
Very small typo in tests. season param should be confidenceLevel

File: packages/sources/blocknative/test/unit/blockprices-eip1559.test.ts (hunk 1 above)
Comment (vnavascues) [reply packages/sources/blocknative/test/unit/blockprices-eip1559.test.ts:19]: Spotted on! Thanks Marc

File: packages/sources/blocknative/test/unit/blockprices-legacy.test.ts (hunk 2 above)
Comment (vnavascues) [reply packages/sources/blocknative/test/unit/blockprices-legacy.test.ts:19]: Spotted on! Thanks Marc

**Review by 1marcghannam — APPROVED** [review]

**Pull request merged**